### PR TITLE
ci: add release script

### PR DESCRIPTION
**Pull request by jerome-quere**

### CODE REVIEW

File: scripts/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (kindermoumoute) [inline scripts/package.json]: move this in `script/js`?

File: scripts/release.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+async function main() {
+
+  //
+  // Initialization
+  //
+
+  console.log("Trying to guess gihub username from git remote origin url (only work with ssh)".blue);
+  const githubUsername = execSync("git", ["remote", "get-url", "origin"]).match(/.*:([a-z0-9-]+)\/.*/)[1];
```
Comment (kindermoumoute) [inline scripts/release.js]: plot twist: my origin is not my gh username + I rarely name a remote with a username. Let's add catch an env variable?

File: scripts/release.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  console.log(`    We found username: ${githubUsername}`.green);
+
+  console.log("Check that remote with name upstream is valid".blue);
```
Comment (kindermoumoute) [inline scripts/release.js]: same here, my upstream is not scaleway's repo

File: scripts/release.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  console.log("Trying to find last release tag".blue);
+  const lastSemverTag = execSync("git", ["tag"])
+    .trim()
+    .split("\n")
+    .filter(semver.valid)
+    .sort((a, b) => semver.rcompare(semver.clean(a), semver.clean(b)))[0];
+  console.log(`    Last found release tag was ${lastSemverTag}`.green);
+
+  console.log("Listing commit since last release".blue);
+  const commits = (await getStream.array(gitRawCommits({ from:  "v1.0.0-beta.2", to: "v1.0.0-beta.3", format: "%s" }))).map(c => c.toString().trim());
```
Comment (kindermoumoute) [inline scripts/release.js]: these commits should not be constant

File: scripts/release.js (hunk 2 above)
Comment (jerome-quere) [reply scripts/release.js]: it parse string like `git@github.com:scaleway/scaleway-sdk-go.git`

Comment (kindermoumoute) [reply scripts/release.js]: For most people yes, but in my case `git remote get-url origin` is `git@github.com:kindermoumoute/scaleway-sdk-go.git`

Comment (jerome-quere) [reply scripts/release.js]: that ok regex in this case will return `kindermoumoute` this is the expected value

File: scripts/release.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
+#!/bin/env node
```
Comment (loicbourgois) [inline scripts/release.js]: Maybe add a short explanation of what this scripts is used for. 
Add a link to public documentation.

File: scripts/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Scaleway GO SDK - Scripts
+
+This directory contain useful scripts to work on scaleway-go-sdk.
```
Comment (kindermoumoute) [inline scripts/README.md]: contains*

File: scripts/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+### check_for_tokens.sh **
+
+Check that no token are present in cassette file
```
Comment (kindermoumoute) [inline scripts/README.md]: Checks

File: scripts/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+```
+Usage: ./script/check_for_tokens.sh
+```
+
+### release.sh **
+
+This script will trigger the release process.
+For more information on the release process you can refer to ./release/release.js file
```
Comment (kindermoumoute) [inline scripts/README.md]: this file MUST be linted

File: scripts/release/release.js [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+ *     - Generate a changelog
+ *     - Update version in scw/version.go
+ *     - Ask to review changes
+ *     - Ask to merge this changes to master via PR
+ *   - Create a github release
+ *     - Ask to create a github release
+ *   - Create a post release commit
+ *     - Update scw/version.go to add +dev at the end
+ *     - Ask to merge this changes to master via PR
+ ******************************************************************************/
```
Comment (kindermoumoute) [inline scripts/release/release.js]: does it delete the new remote?

Comment (jerome-quere) [reply scripts/release/release.js]: I add a line

**Pull request merged**